keyboards/keychron/q1/rev_0100/keymaps/teimor/README.md

```diff
@@ -0,0 +1,13 @@
+# teimor's Keychron Q1 keymap
```

Comment:
Please rename the file to be lower case.

Reply:
done.